### ISSUE TITLE
Cannot instantiate multiple shake detectors

### ISSUE THREAD

**Issue by risharde** [safetysystemtechnology/android-shake-detector#3]
Hi, I love this module but I have a scenario where I'd like multiple shake detectors. For example, to detect when a single shake has occurred vs when a double shake has occurred. I tried instantiating 2 shake detectors but it seems that the second overrides the first.

Not sure if this a bug and if there's a workaround. Would appreciate your assistance.

Thanks

**Comment by netodevel** [comment]
I have not really tested this scenario ..
I will see what I can do
Thank you.

**Comment by svetaz** [comment]
Yup,same results here,2nd one overrides the 1st one.Is there any solution for this?
Thanks in advance.
